sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/OrcRelation.scala

```diff
@@ -120,15 +111,11 @@ private[orc] class OrcOutputWriter(
   }
 
   override def write(row: Row): Unit = {
-    var i = 0
-    while (i < row.length) {
-      reusableOutputBuffer(i) = wrappers(i)(row(i))
-      i += 1
-    }
+    val orcRow = wrap(row, structOI)
```

Comment:
Looks like this call will create a new object for each row written instead of reuse reusableOutputBuffer. Is it a concern?